dts/bindings/gpio/nxp,kinetis-gpio.yaml

```diff
@@ -25,6 +25,11 @@ properties:
       description: required interrupts
       generation: define
 
+    daisy-gpios:
```

Comment:
What is this property suppose to mean?

Reply:
My interpretation is that "daisy" will be the preample for any items that will have a daisy chain of IRQ. In this case, GPIO is daisy chained.